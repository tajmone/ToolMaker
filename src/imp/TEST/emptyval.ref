foo ><, fie ><, fum >Hej<
OK!
OK!
