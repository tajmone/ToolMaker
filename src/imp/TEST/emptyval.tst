%%SET foo()
%%SET fie( )
%%SET fum(Hej)
foo >%%(foo)<, fie >%%(fie)<, fum >%%(fum)<
%%IF(foo() AND fie( ) AND fum())
OK!
%%ELSE
Error!
%%ENDIF
%%IF(NOT foo(*) AND NOT fie(*) AND fum(*))
OK!
%%ELSE
Error!
%%ENDIF
