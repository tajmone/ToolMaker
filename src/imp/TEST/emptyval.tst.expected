########## emptyval ##########
%%SET foo()
%%SET fie( )
%%SET fum(Hej)
foo >%%(foo)<, fie >%%(fie)<, fum >%%(fum)<
%%IF(foo() AND fie( ) AND fum())
OK!
%%ELSE
Error!
%%ENDIF
%%IF(NOT foo(*) AND NOT fie(*) AND fum(*))
OK!
%%ELSE
Error!
%%ENDIF
########## ../imp emptyval.tst >emptyval.tst.tmp; cat emptyval.tst.tmp; rm emptyval.tst.tmp ##########
foo ><, fie ><, fum >Hej<
OK!
OK!
